### PR TITLE
VIH-7711 Auto change audio to yes if there is a interpreter

### PR DESCRIPTION
**Pull request by ZJianWang**
### JIRA link (if applicable) ###
VIH-7711

### Change description ###

Auto change audio to yes when hearing has interpreter as a participant 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite.UnitTests/Mappers/HearingUpdateRequestMapperTest.cs [hunk 1]
```diff
@@ -27,17 +34,52 @@ public void Should_map_properties_for_update_hearing_request()
...
+            result.ScheduledDateTime.Should().Be(_scheduledDateTime);
+            result.ScheduledDuration.Should().Be(_newParticipantRequest.ScheduledDuration);
+            result.OtherInformation.Should().Be(_newParticipantRequest.OtherInformation);
+            result.Cases.Should().BeEquivalentTo(_caseRequest);
             result.QuestionnaireNotRequired.Should().BeFalse();
             result.AudioRecordingRequired.Should().BeFalse();
         }
+
+        [Test]
+        public void Should_Be_true_for_when_linkedParticipant_is_interpreter()
```
Comment (FarronF) [inline AdminWebsite/AdminWebsite.UnitTests/Mappers/HearingUpdateRequestMapperTest.cs:55]: What should be true?

AudioRecordingRequired?

File: AdminWebsite/AdminWebsite.UnitTests/Mappers/HearingUpdateRequestMapperTest.cs [hunk 2]
```diff
@@ -27,17 +34,52 @@ public void Should_map_properties_for_update_hearing_request()
...
+            };
+            var result = HearingUpdateRequestMapper.MapTo(_newParticipantRequest, _username);
+
+            result.HearingRoomName.Should().Be(_newParticipantRequest.HearingRoomName);
+            result.HearingVenueName.Should().Be(_newParticipantRequest.HearingVenueName);
+            result.ScheduledDateTime.Should().Be(_scheduledDateTime);
+            result.ScheduledDuration.Should().Be(_newParticipantRequest.ScheduledDuration);
+            result.OtherInformation.Should().Be(_newParticipantRequest.OtherInformation);
+            result.Cases.Should().BeEquivalentTo(_caseRequest);
+            result.QuestionnaireNotRequired.Should().BeFalse();
```
Comment (FarronF) [inline AdminWebsite/AdminWebsite.UnitTests/Mappers/HearingUpdateRequestMapperTest.cs:81]: I think this should be 

result.QuestionnaireNotRequired.Should().Be(_newParticipantRequest.QuestionnaireNotRequired);

File: AdminWebsite/AdminWebsite.UnitTests/Mappers/HearingUpdateRequestMapperTest.cs (hunk 1 above)
Comment (ZJianWang) [reply AdminWebsite/AdminWebsite.UnitTests/Mappers/HearingUpdateRequestMapperTest.cs:55]: yeah

File: AdminWebsite/AdminWebsite/Controllers/HearingsController.cs [hunk 3]
```diff
@@ -62,7 +64,7 @@ public class HearingsController : ControllerBase
             _conferenceDetailsService = conferenceDetailsService;
             _publicHolidayRetriever = publicHolidayRetriever;
         }
-
+#pragma warning restore S107
```
Comment (kevin-hmcts) [inline AdminWebsite/AdminWebsite/Controllers/HearingsController.cs:67]: Please create a ticket to remove the  "#pragma" directive

**Review by kevin-hmcts — APPROVED** [review]
Please create a ticket to remove the "#pragma" directive

**Pull request merged**